src/Simplex/Chat/Controller.hs

```diff
@@ -223,7 +222,7 @@ data ChatCommand
   | SetTempFolder FilePath
   | SetFilesFolder FilePath
   | APISetXFTPConfig (Maybe XFTPFileConfig)
-  | SetIncognito Bool
+  | HelpIncognito (Maybe Bool)
```

Comment:
better parse it as Help command, ignoring the parameter, then you don't need additional response